cmd/cmp/main.go

```diff
@@ -20,11 +17,15 @@ var (
 )
 
 func Run(_ *cobra.Command, args []string) {
+	returnCode := CompareRun(args)
+	os.Exit(returnCode)
+}
+
+func CompareRun(args []string) int {
```

Comment:
```suggestion
func compareRun(args []string) int {
```

Go use the first letter to indicate the visibility level. Function names start with a capitalized letter will make them public. In this case, we do not need this public.